utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java

```diff
@@ -68,13 +74,53 @@ public void validateLead() {
 		Assertions.assertThat(getLeadTaskFromDb().toLowerCase()).isNotEmpty();
 	}
 
+	@Then("^validate add_lead procedure with last_name: \"([^\"]*)\", company: \"([^\"]*)\", period in ms: \"(\\w+)\"")
+	public void validateAddLeadProcedure(String lastName, String company, Integer ms) {
+		//wait for period cycle:
+		Selenide.sleep(ms + 100);
```

Comment:
We definitely don't need Selenide's sleep, as it brings whole dependency. `Thread.sleep` or `TimeUnit.SECONDS.sleep` is good enough. If you mind the `InterruptedException` add something like sleepQuietly() into `TestUtils` class.